wasmtime/_extern.py

```diff
@@ -1,5 +1,10 @@
 from ._ffi import *
+
 from ctypes import *
+import typing
+
+if typing.TYPE_CHECKING:
+    from ._exportable import Exportable
```

Comment:
I think it's fine to do this unconditionally? Or am I missing something for why we might want this to not happen all the time?

Reply:
Sure. To explain, I treat every import into any namespace as potentially breaking things, or potentially bringing in tonnes of code, without realizing it, that doesn't need to be there at run-time. I'm used to working on large, monolithic machine learning codebases where one unnecessary import could bring Pandas into a small Docker container when it's not needed.

Reply:
That makes sense, but this is all code controlled in this package, so I think we can ensure that we keep a handle on our own imports?